docs/deprecations.md

```diff
@@ -10,7 +10,7 @@ layout: Doc
 
 ## Automatic loading environment variables from .env and .env.{stage} files
 
-Starting with v3.0.0, environment variables will be automatically loaded from `.env` and `.env.{stage}` files if they're present.
+Starting with v3.0.0, environment variables will be automatically loaded from `.env` and `.env.{stage}` files if they're present. In addition, `.env` files will be excluded from package in order to avoid uploading sensitive data as a part of the package by mistake.
```

Comment:
In v3, will there be a way to keep uploading `.env`?

I see the `useDotenv: true` option, but will that affect that part too?

Additionally, is there a way to `useDotenv: false` and keep uploading `.env` files?

Basically, sometimes `.env` files are for the framework, not `serverless`.

Reply:
These are all valid questions and I believe they deserve careful consideration before making a final decision with v3. I guess one of the potential options would be to allow setting `useDotenv` to false in v3, another one that comes to my mind is to introduce a separate, opt-in parameter that would allow including `.env` files if by default they would be excluded. What is your opinion @medikoo @mnapoli? (I'm also not sure if that discussion should be happening here or in related Issue: https://github.com/serverless/serverless/issues/8566)

Reply:
@mnapoli if with `useDotenv` you still want `.env` to be packaged I suggest to simply add:

```yaml
package:
  include: ".env*"
```

I believe that should do the job.

Currently we do not plan to provide support `useDotenv: false`, as there seems no confirmed use case for that.
Still, we're still discussing this at: https://github.com/serverless/serverless/issues/8566

If you feel there's a valid reason to do that please share your thoughts over there.

Reply:
I'll merge the current PR as-is and let's continue further discussion in the mentioned issue :+1:

Reply:
I must say that this `Deprecation warning: Detected ".env" files. Note that Framework now supports loading variables from those files when "useDotenv: true" is set` message is very annoying.

I'm using `.env` file for my local and integration test in `jest`. It is not for the framework at all. In my case, it is a false positive.

Is there a way to turn off this warning?

Reply:
@serverlesspolska Yes, if you do not wish to address deprecation and for some reason want the log message to be gone you can turn it off with `SLS_DEPRECATION_DISABLE`, check https://www.serverless.com/framework/docs/deprecations/

Reply:
Thank you @medikoo for this info. I was not aware. 

However, this is not the best approach this may vary from project to project. 
In my opinion, it would be much better to have this setting explicitly in `serverless.yml` file

Reply:
Actually there's a setting for it in `serlveress.yml` (we simply forgot to add it to document, cc @pgrzesik ). Check `disabledDeprecations`. It's listed here https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/

Still this one will work only for deprecations triggered after config is loaded.

Reply:
I just check it, and it doesn't seem to work 
![image](https://user-images.githubusercontent.com/22025007/104034045-a6e4ec00-51d0-11eb-8675-888d7394da88.png)

Reply:
Please report with new issue

Reply:
@medikoo same behavioru with the second option:
```
disabledDeprecations:
  - LOAD_VARIABLES_FROM_ENV_FILES
```

Even `*` doesn't seem to have any impact.

Reply:
Please report with new issue